src/Http/Http.Abstractions/src/Routing/RouteValueDictionary.cs

```diff
@@ -744,6 +745,7 @@ internal class PropertyStorage
         public readonly object Value;
         public readonly PropertyHelper[] Properties;
 
+        [RequiresUnreferencedCode("PropertyStorage uses reflection to get property values.")]
```

Comment:
```suggestion
```
Will look at this change again in followup PR to resolve Http.Abstractions warnings